src/services/login.ts

```diff
@@ -3,8 +3,8 @@ import ajax from 'services/ajax';
 const gapi = global.gapi;
 gapi.load('auth2');
 
-export const googleLogin = token =>
-  ajax.get('/oauth/google/token', { headers: { token } });
+export const googleLogin = idToken =>
+  ajax.get('/oauth/google/token', { headers: { id_token: idToken } });
```

Comment:
Are you still pointing to the old API? this needs to be `token`

```
REACT_APP_API=https://demo.ego.kfdrc.org/
```

Reply:
ahh yes, I must be.